futurenhs.app/pages/users/[userId]/update/index.page.tsx

```diff
@@ -88,7 +88,7 @@ export const getServerSideProps: GetServerSideProps = withUser({
                      */
                     try {
                         const [userData] = await Promise.all([
-                            getSiteUser({ user, targetUserId }),
+                            getSiteUser({ user, targetUserId, isForEdit: true }),
```

Comment:
could we call this isForUpdate? I'm as guilty as anyone for this, but we have a combination of term uses between update and edit. I'd rather we try and stick to update so it matches the http verb it's usually used with in requests

Reply:
@ReallySmall in the other places we're using this we have isForEdit - group members, groups, folders etc so I followed this pattern. Want me to change them all to isForUpdate?

Reply:
@ReallySmall Addressed this now, changed all instances to isForUpdate instead, matches the route & http verb as you say